gradle/libs.versions.toml

```diff
@@ -0,0 +1,53 @@
+[versions]
+opentelemetry = "1.29.0"
+opentelemetry-alpha = "1.29.0-alpha"
+mockito = "5.5.0"
+junit = "5.10.0"
+byteBuddy = "1.14.8"
+okhttp = "4.11.0"
+spotless = "6.22.0"
+
+[libraries]
+opentelemetry-platform = { module = "io.opentelemetry.instrumentation:opentelemetry-instrumentation-bom-alpha", version.ref = "opentelemetry-alpha" }
```

Comment:
This should be moved to `[bundles]` right?

Reply:
I gave it a try and moved this line below `[bundles]` though I got this error message:

```
* Exception is:
java.lang.RuntimeException: org.gradle.api.InvalidUserDataException: Invalid TOML catalog definition:
  - Unexpected type for bundle 'opentelemetry-platform'
    
    Reason: Expected an array but value of 'opentelemetry-platform' is a table.
    
    Possible solution: Use an array instead.
```

I'm not an expert using this feature of Gradle, however, based on [this part about bundles](https://docs.gradle.org/current/userguide/platforms.html#sec:dependency-bundles), it seems to me that bundles are kind of like buckets to place a bunch of individual dependencies declared under `[libraries]` that usually go together (such as mockito-core and mockito-junit-jupiter), so that you can later add a single dependency line with a bundle in a project's build.gradle, that would be the same as adding all the dependencies listed in that bundle (as it's done [here](https://github.com/LikeTheSalad/opentelemetry-android/blob/central-dependencies/instrumentation/build.gradle.kts#L74) for example), so I think that if we wanted to use bundles with OTel libs, we'd need to declare all of OTel's individual libs under `[libraries]` and then put them all into a bundle that we can add to projects that need all of those libs. However, if we're sticking to using this BOM, the only way I found to mix both, the "platform" dependency with the "catalogs", was to define the platform dependency as a regular one in the catalog, and then wrap it inside `platform()` [in here](https://github.com/LikeTheSalad/opentelemetry-android/blob/central-dependencies/instrumentation/build.gradle.kts#L65).

Reply:
Cool, thank you for giving it some consideration.